examples/multimap/main.go

```diff
@@ -52,5 +52,11 @@ func main() {
 	if err != nil {
 		log.Fatal(err)
 	}
-	fmt.Printf("entry found:%t", ok)
+	fmt.Printf("entry found:%t\n", ok)
+	// Print number of values match given "key"
+	count, err := m.ValueCount(ctx, "key")
+	if err != nil {
+		panic(err)
+	}
+	fmt.Printf("Number of values on \"%s\": %v", "key", count)
```

Comment:
How about "Number of values for key: %s"? Also, you can use backticks in order to not escape the quotes.

Reply:
[4df6cf3](https://github.com/hazelcast/hazelcast-go-client/pull/734/commits/4df6cf34201606cf13235de430fa990af177b711).